examples/albert/run_training_monitor.py

```diff
@@ -139,17 +139,21 @@ def upload_checkpoint(self, current_loss):
     parser = HfArgumentParser((CoordinatorArguments, CollaborativeOptimizerArguments, AveragerArguments))
     coordinator_args, collab_optimizer_args, averager_args = parser.parse_args_into_dataclasses()
 
-    if coordinator_args.address is None:
-        logger.warning("No address specified. Attempting to infer address from DNS.")
-        coordinator_args.address = get_ip(GoogleDnsProvider)
+    if coordinator_args.use_google_dns:
+        address = get_ip(GoogleDnsProvider)
+        logger.info(f"Google DNS responds that our IP address is {address}")
```

Comment:
I would argue that the proposed message is more misleading (looks like the DNS tells us the IP address for some domain since that's what DNS usually does). However, I don't mind changing the message to another, more clear one.

Reply:
What about `"Received IP address of monitor from Google DNS: {address}"`?

Reply:
I'd say `monitor` is an ambiguous term. I've changed to `Received public IP address of this machine from Google DNS`.